mattermost/mattermost-webapp
# MM-49503 - Show in-app Pricing Modal in Edition & License After a License is Uploaded

#### Summary

After purchasing (or uploading a license), we don’t have a way to view the in-app pricing modal.
The Contact sales link only goes to [Pricing | Mattermost](https://mattermost.com/pricing).

We’d like to offer some way to show the in-app pricing modal from the license page once a license is uploaded.

The `View Plans` button is already visible here in some cases (such as for a free workspace).

#### Ticket Link

[MM-49503](https://mattermost.atlassian.net/browse/MM-49503)

#### Screenshots

<img width="1689" alt="Screen Shot 2023-02-03 at 10 57 49 AM" src="https://user-images.githubusercontent.com/116016004/216648762-03f15d75-be6d-4853-8edc-b0f2fbd6331d.png">

#### Test Plan

1. Navigate to `System Console > About > Edition and License.`
2. Upload a license.
3. Ensure the `View plans` button appears beside the license type on the left-hand card.
4. Click the `View plans` button. The pricing modal should appear.
5. The pricing modal is self-contained and should give you the same experience as if you opened it via the `System Console > Billing & Account > Subscription` page.
6. **NOTE**: The plan in the pricing modal is based on the current subscription plan, and NOT the license you uploaded.

#### Release Note

```release-note
NONE
```

[MM-49503]: https://mattermost.atlassian.net/browse/MM-49503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
